src/Eloquent/Traits/BuildsAdjacencyListQueries.php

```diff
@@ -12,6 +14,9 @@
 use Staudenmeir\LaravelAdjacencyList\Query\Grammars\SQLiteGrammar;
 use Staudenmeir\LaravelAdjacencyList\Query\Grammars\SqlServerGrammar;
 
+/**
+ * @mixin \Staudenmeir\LaravelAdjacencyList\Eloquent\Builder
```

Comment:
@calebdw with this mixin PHPStan throws this error:

```
ErrorException thrown in C:\Users\X\Documents\GitHub\laravel-adjacency-list\vendor\laravel\framework\src\Illuminate\Filesystem\Filesystem.php on line 233 while loading bootstrap file C:\Users\X\Documents\GitHub\laravel-adjacency-list\vendor\larastan\larastan\bootstrap.php: rename(C:\Users\X\Documents\GitHub\laravel-adjacency-list\vendor\orchestra\testbench-core\laravel\bootstrap\cache\pacD6F9.tmp,C:\Users\X\Documents\GitHub\laravel-adjacency-list\vendor\orchestra\testbench-core\laravel\bootstrap\cache/packages.php): Access is denied (code: 5)
```

Any idea why? On MacOS it throws another error, see https://github.com/staudenmeir/laravel-adjacency-list/issues/233
Very weird. I solved it kinda, but would still like what's going on, since this error might still exist for other combinations of Relation types and scopes

Reply:
The error you're showing looks like a cache issue, try clearing it with `phpstan clear-result-cache`